ui/src/main/webapp/wizards/sources/reducer.js

```diff
@@ -69,73 +65,34 @@ const isSubmitting = (state = false, { type }) => {
   }
 }
 
-const configTypes = (state = fromJS([]), { type, types }) => {
-  switch (type) {
-    case 'SOURCES/SET_CONFIG_IDS':
-      return fromJS(types)
-    default:
-      return state
-  }
-}
-
 const discoveryType = (state = 'hostnamePort', { type, value }) => {
   switch (type) {
-    case 'SOURCES/DISCOVERY_TYPE/SET':
+    case 'SOURCES/SET_DISCOVERY_TYPE':
       return value
     default:
       return state
   }
 }
 
-export const getConfigTypeById = (state, id) => {
-  const found = getConfigTypes(state).filter((config) => config.id === id)
-  if (found.length > 0) {
-    return found[0].name
-  }
-}
-
-export const getDiscoveryConfigs = (state) => (type) => {
-  const sourceUserName = state.getIn(['wizard', 'config', 'sourceUserName', 'value'])
-  const sourceUserPassword = state.getIn(['wizard', 'config', 'sourceUserPassword', 'value'])
-  switch (type) {
-    case 'hostnamePort':
-      const sourceHostName = state.getIn(['wizard', 'config', 'sourceHostName', 'value'])
-      const sourcePort = state.getIn(['wizard', 'config', 'sourcePort', 'value'])
-      return {
-        sourceUserName,
-        sourceUserPassword,
-        sourceHostName,
-        sourcePort
-      }
-
-    case 'url':
-      const endpointUrl = state.getIn(['wizard', 'config', 'endpointUrl', 'value'])
-      return {
-        sourceUserName,
-        sourceUserPassword,
-        endpointUrl
-      }
-
-    default:
-      return {
-        sourceUserName,
-        sourceUserPassword
-      }
-  }
-}
-
 export const submarine = sub()
-export const getSourceSelections = (state) => submarine(state).get('sourceSelections')
 export const getIsSubmitting = (state) => submarine(state).get('isSubmitting')
 export const getDiscoveryType = (state) => submarine(state).get('discoveryType')
-export const getSourceStage = (state) => submarine(state).get('sourceStage')
+export const setStage = (state) => submarine(state).get('currentStage')
```

Comment:
I renamed `sourceStage` to `currentStage` when I removed the side-pane nav bars and their associated reducers. Also just removed 2 actions below that were no longer being used.